frontend/src/app/logs/logs.component.html

```diff
@@ -1,122 +1,205 @@
-<div class="oib-padded-container">
-  <div class="oib-header-container">
-    <div class="row mb-2">
-      <div class="col d-flex">
-        <div class="oib-block-title">
-          <img class="oib-title-image pt-2" src="/home/logs.svg" />
+@if (this.scopeId != null && this.scopeType != null) {
```

Comment:
I think it won't display the main log page anymore with this settings. Instead, you should display the scopeId and scopeType fields (in the search form) only if these inputs are empty